python/CONTRIBUTING.rst

```diff
@@ -77,10 +77,8 @@ Ready to contribute? Here's how to set up `nessie` for local development.
    Now you can make your changes locally.
 
 5. When you're done making changes, check that your changes pass flake8 and the
-   tests, including testing other Python versions with tox::
+   tests, including testing other Python versions with tox and all docs have been generated::
```

Comment:
nit: double `:` at the end of line?

Reply:
I think `rst` needs to have double `:` at the end of the line 🤔

Reply:
TIL :facepalm: